### PR TITLE
Add types for Writable#_writev() & Duplex#_writev().

### PR DESCRIPTION
**Pull request by jinwoo**
As written in the doc:
https://nodejs.org/dist/latest-v8.x/docs/api/stream.html#stream_writable_writev_chunks_callback

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not provide its own types, and you can not add them.
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/node/index.d.ts [hunk 1]
```diff
@@ -5110,6 +5110,7 @@ declare module "stream" {
             writable: boolean;
             constructor(opts?: WritableOptions);
             _write(chunk: any, encoding: string, callback: Function): void;
+            _writev(chunks: Array<{chunk: any, encoding: string}>, callback: Function): void;
```
Comment (Flarna) [inline types/node/index.d.ts]: You could consider to use `(err?: Error) => void` (also for `write` and in `Duplex`).

Comment (jinwoo) [reply types/node/index.d.ts]: Sounds good. Done.

**Comment by Flarna** [comment]
Your last commit to change `Array<>` to `[]` causes a lint warning. Other CI fails are unrelated in my opinion.

**Comment by jinwoo** [comment]
Reverted the change from Array<> to [].

**Review by Flarna — APPROVED** [review]

**Comment by jinwoo** [comment]
As @Flarna said, the failures are not related to my change.

**Comment by jinwoo** [comment]
Made another commit that makes _writev() optional.

**Pull request merged**